service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -128,12 +155,27 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
         sendNotification(STANDARD_DIRECTION_ORDER_ISSUED_TEMPLATE, email, parameters, reference);
     }
 
-    private void sendNotification(String templateId, String email, Map<String, Object> parameters, String reference) {
+    private void sendNotification(final String templateId,
+                                 final String email,
+                                 final Map<String, Object> parameters,
+                                 final String reference) {
         log.debug("Sending submission notification (with template id: {}) to {}", templateId, email);
         try {
             notificationClient.sendEmail(templateId, email, parameters, reference);
         } catch (NotificationClientException e) {
             log.error("Failed to send submission notification (with template id: {}) to {}", templateId, email, e);
         }
     }
+
+    private void sendNotifications(final String templateId,
```

Comment:
why not just use .forEach on the list of emails to send notifications and use the existing sendNotifications?

Reply:
we could actually...